app/models/book.rb

```diff
@@ -0,0 +1,5 @@
+class Book < ApplicationRecord
+  has_many :reviews
```

Comment:
we can specify destroy behavior by adding `dependent` options
ref: https://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-dependent